pkg/kubectl/cmd/delete.go

```diff
@@ -184,6 +182,11 @@ func (o *DeleteOptions) Complete(f cmdutil.Factory, out, errOut io.Writer, args
 }
 
 func (o *DeleteOptions) Validate(cmd *cobra.Command) error {
+	outputMode := cmdutil.GetFlagString(cmd, "output")
```

Comment:
This looks very wrong.  Should be controlled by print options nil-ing out bad options and then erroring in complete, right?

Reply:
Delete does not use the PrintFlags struct, so I was not able to to do this here